src/app/shared/rss-feed/rss.component.spec.ts

```diff
@@ -96,17 +96,17 @@ describe('RssComponent', () => {
     });
 
     it('should formulate the correct url given params in url', () => {
-        const route = comp.formulateRoute(uuid, 'opensearch', options, query);
+        const route = comp.formulateRoute(uuid, 'opensearch/search', options, query);
         expect(route).toBe('/opensearch/search?format=atom&scope=2cfcf65e-0a51-4bcb-8592-b8db7b064790&sort=dc.title&sort_direction=DESC&query=test');
```

Comment:
@nwoodward I have created another related bug https://github.com/DSpace/dspace-angular/issues/1935
maybe this can be solved as part of this PR?

Reply:
Yes, I think we can do that. If I understand you correctly, we need to start by removing the sort by title asc in the link coming from the UI and let the default in the REST API (dc.date.accessioned_dt desc) take over?

Reply:
@nwoodward : Yes, I think your understanding is correct.  I tested this PR today, and it always seems to return an alphabetical (by title) RSS/Atom feed (on homepage, Communities, Collections).  That's obviously odd behavior as an RSS feed is supposed to match with the list of *recent submissions*.